tests/fsharp/SDKTests/tests/DefaultFSharpCore - IsNotPreviewSdkDisablePreviewCheck.proj

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" DefaultTargets="Test" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Verifies that in an RTM sdk with the previewcheck disabled it selects the already RTM fsharp.core package -->
```

Comment:
Comment says RTM sdk, but the property 2 lines down indicates it's a preview SDK.